.github/workflows/build_steps.yml

```diff
@@ -120,22 +120,22 @@ jobs:
           buildPreset: ${{env.ARCTIC_CMAKE_PRESET}}
 
       - name: Compile C++ tests
-        if: inputs.job_type == 'cpp-tests'
+        if: inputs.job_type != 'follower'
         run: cd cpp; cmake --build --preset $ARCTIC_CMAKE_PRESET --target install
 
-      - name: Clean up build dir # To save disk space as we're close to the limit on Linux
-        if: inputs.job_type == 'cpp-tests' && matrix.os == 'linux'
-        run: |
-          cd $ARCTICDB_BUILD_DIR/*-build
-          du -m --max-depth=2 | sort -n | tail -n 50
-          nohup rm -rf * &  # Clean up while the test is running to save build time
+      # - name: Clean up build dir # To save disk space as we're close to the limit on Linux
+      #   if: inputs.job_type != 'follower' && matrix.os == 'linux'
+      #   run: |
+      #     cd $ARCTICDB_BUILD_DIR/*-build
+      #     du -m --max-depth=2 | sort -n | tail -n 50
+      #     nohup rm -rf * &  # Clean up while the test is running to save build time
```

Comment:
Do not commit commented-out code.

You can add a condition to the `if` that's always false or remove it.